querydsl-core/src/test/java/com/querydsl/core/types/TemplateTest.java

```diff
@@ -31,6 +31,20 @@ public void test() {
     }
 
     @Test
+    public void operation() {
+        TemplateFactory factory = new TemplateFactory('\\');
+        match("[0 ADD 1]",               factory.create("{0+1}"));
+        match("[0 DIV 1]",               factory.create("{0/1}"));
+    }
+
+    @Test
+    public void operationConst() {
+        TemplateFactory factory = new TemplateFactory('\\');
+        match("[0 ADD 1]",               factory.create("{0+'1'}"));
```

Comment:
The separate `1` is an argument with _index 1_.
The `'1'` is a _constant 1_, which could also have been `'33'` for example.